.travis.yml

```diff
@@ -0,0 +1,29 @@
+cache:
+  directories:
+    - $HOME/.m2
+
+services: docker
+
+language: java
+
+install: skip
+
+stages:
+  - test
+  - package
+  - install
```

Comment:
Shouldn't it be the first step? Can we run the tests before the installation step?

Reply:
Maven "install" goal is after "package". Originally, the "install" stage only installs the generated package on .m2 repository, but in our project it also deploys the contract files on [contracts](https://github.com/caelum-airlines/contracts) repository through Spring Contractor Maven Plugin, thus the necessity to create a specific stage for this goal.